### ISSUE TITLE
Pause at TOD doesn't trigger when thrust is not in managed mod

### ISSUE THREAD

**Issue by hargn** [headwindsim/aircraft#77]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2023-10-11T10:49:57+00:00",
    "ref": "refs/tags/v0.5-alpha.11",
    "sha": "590518f681fe2d24cb06df6b8c02a8398f4b4e1f",
    "actor": "Revyn112",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.5-alpha.11",
    "version": "v0.5.0-HEAD.590518f"
}
```

### Describe the bug

Two times in different flights the A339 didn't respect the Pause at TOD :
I've found it still flying at cruise level after to have overpass the last flightplan waypoint in the last heading it had the time it reached it.

After some tests, i've figured that the Pause at TOD doesn't trigger when the autothrust has not be enforced in managed mod.

### Expected behavior

When the aircraft reach the point on flightplan, the distance it has been set in the EFB from the TOD point and the AP is on, the aircraft should pause the simulation, whatever the state of knobs on the FCU.
Pause at TOD should trigger if the autothrust is in managed mod or in selected mod.

### Steps to reproduce

1. Take the aircraft in cruise level, far away the TOD, with the AP on, and the autothrust in 
2. Setup in the EFB the Pause at TOD feature, at a random distance (at it seems to don't matter)
3. Disconnect the AP : Pause is triggered.
4. Click OK and reconnect the AP
5. **Disconnect the AP : Pause don't trigger**
6. Reconnect the AP and push the Speed Selector knob to enforce the managed speed
7. Disconnect the AP : Pause trigger as it should

### Discord Username (optional)

alwiswennerstroem

**Comment by Revyn112** [comment]
@hargn can you test this behaviour in the a32nx aswell?

**Comment by hargn** [comment]
Yes i'll test, but not today : i'm on a OMDB-WSSS flight with the HW A339 on which i've performed my tests.

**Comment by hargn** [comment]
Related FBW A32NX issue : https://github.com/flybywiresim/aircraft/issues/8261
